apps/web/src/state/farmsV4/state/extendPoolsAtom.ts

```diff
@@ -29,24 +29,27 @@ export type ExtendPoolsQuery = {
 //   before: '',
 //   after: '',
 // })
+//
+
+const poolsAtom = atom([] as PoolInfo[])
 
 export const extendPoolsAtom = atom(
-  [] as PoolInfo[],
-  async (get, set, query: ExtendPoolsQuery & { reset?: boolean }): Promise<PoolInfo[]> => {
+  (get) => get(poolsAtom),
```

Comment:
i change this atom to getter and setter, the setter for fetch pool list dynamic, this may the better way for me to fetch pool list when the existing farm/pool data consumed

Reply:
cc @ChefJerry